src/ILCompiler.Compiler/src/ILCompiler.Compiler.csproj

```diff
@@ -95,9 +98,13 @@
     <Compile Include="..\..\Common\src\Internal\Runtime\EETypeBuilderHelpers.cs">
       <Link>Common\EEType.cs</Link>
     </Compile>
+    <Compile Include="..\..\Common\src\Internal\Runtime\ModuleHeaders.cs">
+      <Link>Common\ModuleHeaders.cs</Link>
+    </Compile>
     <Compile Include="..\..\Common\src\Internal\NativeFormat\NativeFormatWriter.Primitives.cs">
       <Link>Common\NativeFormat\NativeFormatWriter.Primitives.cs</Link>
     </Compile>
+    <Compile Include="$(BaseIntermediateOutputPath)\Native\$(BinDirOSGroup).$(BinDirPlatform).$(BinDirConfiguration)\Runtime\Full\AsmOffsets.cs" />
```

Comment:
+1. We should not be including any platform-specific runtime .cs files in the compiler.